denonavr/denonavr.py

```diff
@@ -218,6 +219,12 @@ def get_command(self, request: str) -> str:
     def send_get_command(self, request: str) -> str:
         """Send HTTP GET command to Denon AVR receiver...for compatibility."""
 
+    def register_callback(self, callback: lambda *args: any):
```

Comment:
```suggestion
    def register_callback(self, type: str, callback: Callable[[str, str], Awaitable[None]]):
```
Why not giving everybody all the options?

Reply:
Great point. I was originally thinking specifically of how this would be used in HA which is shortsighted.